lib/matplotlib/tests/test_cycles.py

```diff
@@ -27,6 +27,17 @@ def test_marker_cycle():
     assert [l.get_marker() for l in ax.lines] == ['.', '*', 'x', '.']
 
 
+@pytest.mark.parametrize('marker', [
+    ['+', mpl.markers.CARETRIGHT],
+    ['*', '.', 'x'],
+    [1, 2, 3],
+    [1, "+", ".", 4],
+])
```

Comment:
Is there any particular need to have 4 cycles instead of 1?

Reply:
Not particularly, the last test case `[1, "+", ".", 4]` should suffice with testing both strings and integers.